src/main/java/com/jc/tm/database/dao/TaskDao.java

```diff
@@ -0,0 +1,23 @@
+package com.jc.tm.database.dao;
+
+import com.jc.tm.database.entity.Task;
+
+import java.util.List;
+
+public interface TaskDao {
+
+    //create or insert task
+    void insetTask(Task task);
+
+    //update task
+    void updateTask(Task task);
+
+    //select task
+    Task selectTask(int id);
+
+    //select all task
+    List<Task> selectAllTask();
```

Comment:
just allTask, but this method is described in not proper way, because we should to have pagination